eelbrain/_experiment/epochs.py

```diff
@@ -508,7 +509,7 @@ def decim_param(
 
     if samplingrate is not None:
         decim_ratio = info['sfreq'] / samplingrate
-        if decim_ratio % 1:
+        if not math.isclose(decim_ratio % 1, 1, rel_tol=1e-3):
```

Comment:
I think you meant:
```suggestion
        if not math.isclose(decim_ratio % 1, 0, rel_tol=1e-3):
```
Also, if we allow slight discrepancies, we should probably round before taking `int()` below?

Reply:
Sorry for the mistake. Changed in https://github.com/zhangwenchili/Eelbrain/pull/14/commits/a8fa5b831b595fbed5fa8fd1dc2e4ddc6009f36c. Yes, we should take round instaead of modulus, because the modulus result can be close to 1 or 0 and we want both.